### ISSUE TITLE
Temporarily locked account

### ISSUE THREAD

**Issue by itssmaxy** [InstaPy/InstaPy#5804]
Alright so my account has gotten temporarily locked for the second time now. I unlock it by gettting a verification code sent to me. Now my question is; is it harmless to keep getting your account temporarily locked? Is there any upper limit on how many times your account can get locked before ban?

**Comment by shubhayu-64** [comment]
It is not harmful to have temporary locks as long as you keep unlocking with proper verification. It is just that the Instagram algorithm finds your activity suspicious and flags your account for temporary ban. 

It does get a lot irritating at times but it dosn't harm your account as such. Also you can temporarily stop using InstaPy to get things back to normal.

**Comment by itssmaxy** [comment]
Thanks for your reply! 
How long would you recommend to take a break from instapy? I’ve taken a few days off but apparently that wasn’t enough...

**Comment by shubhayu-64** [comment]
We actually can't comment on that as you might come back after a week and use instapy again and instagram might again flag you. I'll suggest you take a break for a week or so and then use instapy in a very low frequency. Like try to reduce the bot like activity. You can increase the time gap between two successive likes or follows. Just slow things down.

**Comment by shubhayu-64** [comment]
@itssmaxy could you please assign this issue to me?

**Comment by itssmaxy** [comment]
> @itssmaxy could you please assign this issue to me?

Not sure I have the privileges to do that?

**Comment by Teknival** [comment]
Hi there, yesterday i've tried instapy, after 30 likes with the basic quickstart.py instagram block my like for the respect of community or whatever.

We need a solution please, i'm a noob in programming and on python too lol, if anybody have a script to bypass the instagram block it would be the best add on we can do on instapy 👍🏻 

Thank you @timgrossmann 💯

**Comment by alexal1** [comment]
@itssmaxy @Teknival I faced the same problem and has found a solution, guys!

My account was kept locking each time I used InstaPy. I think that's because Instagram is very suspicious to browser actions now. You see, InstaPy has thousands of users and I'm sure Instagram is aware of it.

But what if you automate actions directly on your phone? Instagram can't block people just because they spend much time using Instagram app in their phones 🙂

So, I created Insomniac (https://github.com/alexal1/Insomniac). You're welcome!
